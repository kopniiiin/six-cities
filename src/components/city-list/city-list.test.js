import React from "react";
import {shallow} from "enzyme";

import CityList from "./city-list.jsx";

import testMocks from "../../test-mocks/city-list.js";

describe(`snapshot test: CityList component`, () => {
  it(`should render correctly`, () => expect(
      shallow(<CityList {...testMocks}/>)
  ).toMatchSnapshot());
});
