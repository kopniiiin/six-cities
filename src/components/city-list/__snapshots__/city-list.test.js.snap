// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: CityList component should render correctly 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="tabs__list locations__list"
    >
      <li
        className="locations__item"
        key="city"
      >
        <a
          className="tabs__item tabs__item--active locations__item-link"
          onClick={[Function]}
        >
          <span>
            city
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
